config/locales/shared/registrations.en.yml

```
@@ -1,6 +1,32 @@
 en:
   shared:
     registrations:
+      finance_details:
+        finance_information:
+          lower_tier_message: "Lower tier registration - charge are not applicable."
```

Just confirmed this with @glancyea . Just a typo

```suggestion
          lower_tier_message: "Lower tier registration - charges are not applicable."
```

---

Cheers! :)